README.md

```diff
@@ -270,3 +270,42 @@ These should match the relevant column type. Mapped variables are substituted in
 
 A `variables` map usually isn't needed for simple queries. The basic condition string should automatically get converted to a meaningful type, but when this fails 
 replacing tricky elements with a variable may help.
+
+### Remote registration [Experimental]
+
+For OMERO installations which support TileDB as the OMERO.tables backend it 
```

Comment:
As just discussed, I would be explicit about the fact this feature currently is targeted at OMERO Plus installations.